docs/SUMMARY.md

```diff
@@ -117,6 +117,8 @@
   * [Python feature server](reference/feature-servers/python-feature-server.md)
   * [\[Alpha\] Go feature server](reference/feature-servers/go-feature-server.md)
   * [\[Alpha\] AWS Lambda feature server](reference/feature-servers/alpha-aws-lambda-feature-server.md)
+  * [Remote Offline Store ](reference/feature-servers/remote-offline-store.md)
```

Comment:
I think it would be more intuitive to have two pages in the docs for it
- remote-offline-store.md for the client under Offline Stores - how to configure the client
- offline-feature-server.md for the server under Feature Servers - how to configure the server

They should of course link to one another

Reply:
updated accordingly. Please give another look

Reply:
Looks good, thanks